posts/pytables-b2nd-slicing.rst

```diff
@@ -44,22 +46,22 @@ With the benchmark's default 10x25x50x50 chunk shape, and after experimenting wi
+The optimized b2nd slicing of PyTables already provides some speedups (although not that impressive) in the inner dimensions, in comparison with flat slicing based on the HDF5 filter pipeline (which performs similarly for PyTables and h5py). As explained in `Blosc2 Meets PyTables`_, HDF5 handling of chunked datasets favours big chunks that reduce in-memory structures, while Blosc2 can further exploit parallel threads to handle the increased number of blocks. Our CPU's L3 cache is 36MB big, so we may still grow the chunksize to reduce HDF5 overhead (without hurting Blosc2 parallelism).
 
 Let us raise the chunkshape to 10x25x150x100 (28.6MB) and repeat the benchmark (again with 6 Blosc2 threads):
 
 .. image:: /images/pytables-b2nd-slicing/b2nd_getslice_big.png
   :width: 75%
   :align: center
 
-Much better! Choosing a better chunkshape not just provides up to 5x speedup for the PyTables optimized case, it also results in 3x-4x speedups compared to the performance of the HDF5 filter pipeline.
+Much better! Choosing a better chunkshape not just provides up to 5x speedup for the PyTables optimized case, it also results in 3x-5x speedups compared to the performance of the HDF5 filter pipeline.
 
 Conclusions and future work
 ---------------------------
 
 The benchmarks above show how optimized Blosc2 NDim's two-level partitioning combined with direct HDF5 chunk access can yield considerable performance increases when slicing multi-dimensional Blosc2-compressed arrays under PyTables. However, the usual advice holds to invest some effort into fine-tuning some of the parameters used for compression and chunking for better results. We hope that this article also helps readers find those parameters.
 
-It is worth noting that these techniques still have some limitations: they only work with contiguous slices (that is, with step 1 on every dimension), and on datasets with the same byte ordering as the host machine. Also, although results are good indeed, there may still be room for implementation improvement: for instance, the case of PyTables flat slicing via HDF5 filters (no b2nd) still looks strangely slow in comparison with the equivalent h5py's access; these future enhancements might as well carry over to the b2nd case for even better results.
+It is worth noting that these techniques still have some limitations: they only work with contiguous slices (that is, with step 1 on every dimension), and on datasets with the same byte ordering as the host machine. Also, although results are good indeed, there may still be room for implementation improvement, for instance with extra code profiling and parameter adjustments.
```

Comment:
suggestion: "for instance with extra code profiling and parameter adjustments" -> "but that will require extra code profiling and parameter adjustments"